.github/workflows/build.yml

```diff
@@ -223,7 +223,7 @@ jobs:
         id: cmake_test
         run: |
           cd build
-          ctest -L main --verbose --timeout 900
```

Comment:
@ggerganov I think it does not hurt to run `ggml_debug` on stories260K on each ubuntu build, but please double check